docs/meshstack.configuration.md

```diff
@@ -68,7 +68,16 @@ meshStack assigns a default quota to newly registered [meshCustomers](./meshclou
 }
 ```
 
-[meshPartners](./administration.index.md) can change these quotas for each of their managed meshCustomers individually using the [administration area](administration.customers.md#customer-quota-management).
```

Comment:
Warum hast du das denn gelöscht und was hat das mit der Feedback mailbox zu tun?

Reply:
Es hat in der Configuration reference nicht direkt was zu suchen, aber ja klar grundsätzlich ist der querverweis nicht falsch. An anderer Stelle haben wir ja auch schon querverweise. Ich nehem es wieder rein